python/packages/aws-lambda-sdk/pyproject.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 authors = [{ name = "serverlessinc" }]
 requires-python = ">=3.7"
 dependencies = [
-    "serverless-sdk~=0.1",
+    "serverless-sdk~=0.1.0",
```

Comment:
Is this update needed?

Reply:
Yes, otherwise it installs v0.2 which breaks the tests for lambda sdk.

Reply:
Ok, so same way we should restrict sdk-schema